test/test-api-getreport.js

```diff
@@ -1,11 +1,11 @@
 'use strict';
 
-// Testcase for returning NodeReport as a string via API call
+// Testcase for returning report as a string via API call
 
 const common = require('./common.js');
 const tap = require('tap');
 const nodereport = require('../');
 var report_str = nodereport.getReport();
 common.validateContent(report_str, tap, {pid: process.pid,
-  commandline: [process.argv0, 'test/test-api-getreport.js'].join(' ')
-});
\ No newline at end of file
+  commandline: [process.argv[0], 'test/test-api-getreport.js'].join(' ')
```

Comment:
I think the best option would be to assume the test path was the direct relative path to the test. You should be able to use `process.title` instead of `process.argv0` (I couldn't get this to not work, but I didn't try windows, YMMV).

```diff
diff --git a/test/test-api-getreport.js b/test/test-api-getreport.js
index a36192e..5fdfab8 100644
--- a/test/test-api-getreport.js
+++ b/test/test-api-getreport.js
@@ -5,7 +5,9 @@
 const common = require('./common.js');
 const tap = require('tap');
 const nodereport = require('../');
+
+const testPath = process.argv[1].substr(process.cwd().length + 1);
 var report_str = nodereport.getReport();
 common.validateContent(report_str, tap, {pid: process.pid,
-  commandline: [process.argv0, 'test/test-api-getreport.js'].join(' ')
-});
\ No newline at end of file
+  commandline: [process.title, testPath].join(' ')
+});

```

Obviously that wouldn't handle things like `node $PWD/test-api-getreport.js`, or weird things like `node test/../test/test-api-getreport.js`, but it should hopefully handle windows and manually running the tests (i.e. all common situations).